pkg/imager/download.go

```diff
@@ -1,16 +1,20 @@
 package imager
 
 import (
+	"errors"
 	"io"
 	"net/http"
 	"os"
 	"sync"
 	"sync/atomic"
 
 	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
 )
 
-func Download(log logrus.FieldLogger, url, dst string, total, current *int64) error {
+var errDownloadCanceled = errors.New("Download canceled")
```

Comment:
Could you please convert it to lowercase?

Reply:
It was an issue, actually. Great job with catching it so quickly. The error is returned in different place (from the http request) and was not checked. I moved checking for context cancellation one level upper, looks like it's working properly now.